lib/git_reflow/git_server/pull_request.rb

```diff
@@ -156,54 +158,56 @@ def commit_message_for_merge
           message << "\nLGTM given by: @#{lgtm_authors.join(', @')}\n"
         end
 
-        message
+        "#{message}\n"
       end
 
       def cleanup_feature_branch?
-        GitReflow::Config.get('reflow.always-cleanup') == "true" || (ask "Would you like to cleanup your feature branch? ") =~ /^y/i
+        GitReflow::Config.get('reflow.always-cleanup') == "true" || (ask "Would you like to push this branch to your remote repo and cleanup your feature branch? ") =~ /^y/i
       end
 
       def deliver?
         GitReflow::Config.get('reflow.always-deliver') == "true" || (ask "This is the current status of your Pull Request. Are you sure you want to deliver? ") =~ /^y/i
       end
 
       def cleanup_failure_message
-        GitReflow.say "Cleanup halted.  Local changes were not pushed to remote repo.".colorize(:red)
+        GitReflow.say "Cleanup halted.  Local changes were not pushed to remote repo.", :deliver_halted
         GitReflow.say "To reset and go back to your branch run \`git reset --hard origin/#{base_branch_name} && git checkout #{feature_branch_name}\`"
       end
 
       def merge!(options = {})
-        if deliver? 
+        if deliver?
```

Comment:
No gem :smile:. I use these wonderful magic vim macros to scrub out unwanted white space and tabs:

```
" Make F5 and F6 magic white space scrubbers
nnoremap <silent> <F5> :let _s=@/<Bar>:%s/\s\+$//e<Bar>:let @/=_s<Bar>:nohl<CR>
nnoremap <silent> <F6> :retab<CR>
```